src/kafka-ui-dont-modify-temporay.json

```diff
@@ -282,5 +282,8 @@
       "preparing_rebalance_state": "PreparingRebalance",
       "unknown_state": "Unknown"
     }
+  },
+  "settings": {
```

Comment:
Move this translation string in ui-component repo.  Later we will remove all the translations files as we are moving components in ui-components.